tests/acceptance/features/bootstrap/WebDav.php

```diff
@@ -4940,8 +4941,11 @@ public function findEntryFromPropfindResponse(
 		?string $user = null,
 		string $type = "files"
 	) {
+		$trimmedEntryNameToSearch = '';
 		// trim any leading "/" passed by the caller, we can just match the "raw" name
-		$trimmedEntryNameToSearch = \trim($entryNameToSearch, "/");
+		if ($entryNameToSearch) {
```

Comment:
```suggestion
		if ($entryNameToSearch !== null) {
```

I am scared that `$entryNameToSearch` will be "0" some day and evaluate to `false`